pkg/api/api.go

```diff
@@ -213,6 +216,7 @@ type ExtraOptions struct {
 	FeedFactory      feeds.Factory
 	Post             postage.Service
 	PostageContract  postagecontract.Interface
+	StakingContract  stakingcontract.Contract
```

Comment:
Why not keep the naming consistent?

contract repeating twice doesnt look or sound good. It would be better to call the `Contract` interface as `Interface`, similar to the postage package. This way it would read `stakingcontract.Interface` which makes more sense.